src/calculateRentalCost.js

```diff
@@ -5,6 +5,16 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const sum = 40 * days;
+
+  if (sum < 120) {
+    return sum;
+  } else if (sum >= 280) {
+    return sum - 50;
+  } else if (sum >= 120) {
+    return sum - 20;
```

Comment:
The conditions in the `if` statements are using the total sum instead of the number of days to determine the discount. You should compare the `days` variable against constants that represent the duration thresholds for discounts, not the sum. Additionally, the discount values should also be represented by constants, as per the checklist.